apps/nextjs-website/src/editorialComponents/BannerLink/BannerLink.tsx

```diff
@@ -40,7 +41,8 @@ export const BannerLink = (props: BannerLinkProps) => {
                 },
               }}
             >
-              <div dangerouslySetInnerHTML={{ __html: body }} />
+              {/* Purify the body if it is a string or show the Element */}
+              {typeof body === 'string' ? DOMPurify.sanitize(body) : body}
```

Comment:
I don't think this change is part of this process, I would do it in a separate branch